x/committee/types/codec.go

```diff
@@ -0,0 +1,57 @@
+package types
+
+import (
+	"github.com/cosmos/cosmos-sdk/codec"
+	"github.com/cosmos/cosmos-sdk/x/distribution"
+	"github.com/cosmos/cosmos-sdk/x/gov"
+	"github.com/cosmos/cosmos-sdk/x/params"
+)
+
+// ModuleCdc generic sealed codec to be used throughout module
+var ModuleCdc *codec.Codec
+
+func init() {
+	cdc := codec.New()
+	RegisterModuleCodec(cdc)
+	ModuleCdc = cdc.Seal()
+}
+
+// TODO decide if not using gov's Content type would be better
```

Comment:
do the proposals of this module need to be duplicated?

Reply:
The committee proposal interface type is the same as the one gov uses for proposals. So any gov proposal also works as a committee proposal. Though they need to be registered on the gov module cdc to allow them to be marshalled in gov submit proposal msgs.